packages/origin-247-certificate/src/offchain-certificate/synchronize/handlers/claim-persist.handler.ts

```diff
@@ -42,4 +39,29 @@ export class ClaimPersistHandler implements PersistHandler {
             });
         }
     }
+
+    async handleBatch(events: CertificateEventEntity[]): Promise<void> {
+        const claimedEvents = events as CertificateClaimedEvent[];
+        const result = await this.certificateService.batchClaim(
```

Comment:
So batching needs to handle transaction block size limit.
The safest way is to use this configuration, which works for transfer module: https://github.com/energywebfoundation/origin-247-sdk/blob/master/packages/origin-247-transfer/src/batch/configuration.ts
it can be configurable via env viariables as well but it's not super important

Reply:
fixed